yarnitor/api.py

```diff
@@ -19,7 +19,11 @@ def get_model():
 
 @api_bp.route('/api/status')
 def status():
-    return jsonify({'status': 'ok'})
+    ym = get_model()
```

Comment:
There's a model of the yarn info in the worker and another model in the web stack of the summarized data in redis. They're independent.

Fetching a singleton instance in a route handler is a common flask pattern. Or am I missing what you're seeing as a workaround?